src/app/(proper_react)/(redesign)/(public)/how-it-works/components/SectionOne.tsx

```diff
@@ -0,0 +1,155 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import Image from "next/image";
+import { Props } from "../HowItWorksView";
+import {
+  RemoveStep1,
+  RemoveStep2,
+  RemoveStep3,
+  NameIcon,
+  LocationIcon,
+  DOBIcon,
+} from "../images";
+import styles from "../HowItWorksView.module.scss";
+import { TelemetryButton } from "../../../../../components/client/TelemetryButton";
+
+export const SectionOne = (props: Props) => {
+  const { l10n } = props;
+
+  return (
+    <div
+      id="data-brokers-section"
+      className={`${styles.sectionWrapper} ${styles.grayBG}`}
+    >
+      <div className={styles.intro}>
+        <span className={styles.introContent}>
+          <h2>
+            {l10n.getFragment("section-1-intro-title", {
+              elems: {
+                data_brokers: <span className={styles.brokersEmphasis} />,
+              },
+            })}
+          </h2>
+          <p>{l10n.getString("section-1-intro-text-1")}</p>
+          <hr className={styles.horizontalRule} />
+        </span>
+        <div className={styles.introCTA}>
+          <h3>{l10n.getString("section-1-intro-text-2")}</h3>
+          <div className={styles.sectionCTAButton}>
+            <TelemetryButton
+              variant="primary"
+              href="https://accounts.firefox.com/subscriptions/products/prod_OiV9RSaatywSRy?plan=price_1Nv4ODJNcmPzuWtRoYpoFHXd"
```

Comment:
This might not work for testing on local/dev/stage.

Reply:
Updated here:
https://github.com/mozilla/blurts-server/pull/4632/commits/ef2a5e9f3c71dcf7b0cbf9904c147531bdd01fdd